rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +99,24 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+* If a `Set` RPC contains `origin` CLI, the CLI origin MUST be given precedence
```

Comment:
Just to be clear, "precedence" in this case means not that the CLI commands should precede the OC updates, but rather that any value changed by both a CLI command and an OC update should take on the value set by the CLI command?

And is this the same as saying "all OC updates should be applied first, then all CLI updates",  i.e. the sequence

    cli command 1
    oc update 1
    cli command 2
    oc update 2

should behave exactly equivalently to

    oc update 1
    oc update 2
    cli command 1
    cli command 2
?

Reply:
Precedence is intended to be interpreted as you described.  

Yes, a setRequest with either of the above sequences of updates should have the same effect on the device configuration.